src/components/modules/blockEvents.ts

```diff
@@ -167,30 +167,30 @@ export default class BlockEvents extends Module {
    *
    * @param {ClipboardEvent} event - clipboard event
    */
-  public handleCommandC(event: ClipboardEvent): void {
+  public handleCommandC(event: ClipboardEvent): Promise<void> {
     const { BlockSelection } = this.Editor;
 
     if (!BlockSelection.anyBlockSelected) {
       return;
     }
 
     // Copy Selected Blocks
-    BlockSelection.copySelectedBlocks(event);
+    return BlockSelection.copySelectedBlocks(event);
   }
 
   /**
    * Copy and Delete selected Blocks
    *
    * @param {ClipboardEvent} event - clipboard event
    */
-  public handleCommandX(event: ClipboardEvent): void {
+  public async handleCommandX(event: ClipboardEvent): Promise<void> {
     const { BlockSelection, BlockManager, Caret } = this.Editor;
 
     if (!BlockSelection.anyBlockSelected) {
       return;
     }
 
-    BlockSelection.copySelectedBlocks(event);
+    await BlockSelection.copySelectedBlocks(event);
```

Comment:
Fix the weakness of block cut program. 
If `await` is used in `copySelectedBlocks`, 
`removeSelectedBlocks` is executed before `copySelectedBlocks`